src/app/interactive/page.js

```diff
@@ -35,14 +35,22 @@ const sections = [
 
 export default function IFRCReportPage() {
 	const [activeMenu, setActiveMenu] = useState("toc")
-	const [bookmarkedSections, setBookmarkedSections] = useState(getBookmarks());
+	const [bookmarkedSections, setBookmarkedSections] = useState(new Set());
```

Comment:
```suggestion
   const bookmarks = use(getBookmarks());
	const [bookmarkedSections, setBookmarkedSections] = useState(new Set());
```
What's wrong with 'use'? https://react.dev/reference/react/use

Reply:
Thanks for the clarification! You are right that use() can work with <Suspense />, but for this case useEffect with useState already covers the async client logic cleanly, so would just leave it as it is.